src/v2/Components/ArtworkFilter/ArtworkFilterContext.tsx

```diff
@@ -183,6 +183,8 @@ export interface ArtworkFilterContextProps {
   /** The current artwork filter state (which determines the network request and the url querystring) */
   filters?: ArtworkFiltersState
 
+  defaultFilters?: ArtworkFiltersState
```

Comment:
🙏 for the PR comment, but could we add a comment above line 186 that describes what this does? 

Also wondering if after the A/B test is done we can remove this prop?

Reply:
> if after the A/B test is done we can remove this prop

I think yes

Reply:
Since this prop adds a somewhat large amount of complexity to the artwork filter we should comment around the key areas that can be removed after the test is over

Reply:
Fixed